external-builds/pytorch/README.md

```diff
@@ -496,11 +496,29 @@ have proper defaults.
 You are welcome to maintain your own branches that extend one of AMD's.
 Change origins and tags as appropriate.
 
-#### ROCm release v2.7.x branch
+##### ROCm release v2.9.x branch
 
-NOTE: Presently broken at runtime on a HIP major version incompatibility in the
-pre-built aotriton (#1025). Must build with
-`USE_FLASH_ATTENTION=0 USE_MEM_EFF_ATTENTION=0` until fixed.
```

Comment:
well hopefully fixed soon.. but not yet

Reply:
That (https://github.com/ROCm/TheRock/issues/1025) was referring to an old issue in ROCm which affected PyTorch 2.7, which has since been resolved.